setup.py

```diff
@@ -1,17 +1,18 @@
 #!/usr/bin/env python
 
 from setuptools import setup, find_packages
+
 import provider
 
 setup(
-    name='django-oauth2-provider',
+    name='edx-django-oauth2-provider',
```

Comment:
I have the same question as @nedbat.  How does this change affect existing edX installations?